.swiftlint.yml

```diff
@@ -1,9 +1,8 @@
 disabled_rules:
   - line_length
   - type_name
-  - valid_docs
```

Comment:
> Loading configuration from '.swiftlint.yml'
'variable_name' rule has been renamed to 'identifier_name' and will be completely removed in a future release.
configuration error: __'valid_docs'__ is not a valid rule identifier